### PR TITLE
[ACS-6071] fix jsdoc warnings and errors

### PR DESCRIPTION
**Pull request by DenysVuika**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [x] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

fix jsdoc warnings and errors prior to enabling global jsdoc linter

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:
https://alfresco.atlassian.net/browse/ACS-6071

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -24,7 +24,8 @@ module.exports = {
                 'plugin:@cspell/recommended',
                 'plugin:@angular-eslint/ng-cli-compat',
                 'plugin:@angular-eslint/ng-cli-compat--formatting-add-on',
-                'plugin:@angular-eslint/template/process-inline-templates'
+                'plugin:@angular-eslint/template/process-inline-templates',
```
Comment (AleksanderSklorz) [inline .eslintrc.js:27]: FYI we can easily prevent checking spec files (or other files which we don't want to include) if we do something like suggested here: https://github.com/Alfresco/alfresco-ng2-components/pull/8942#discussion_r1340049673

But we can leave it for now.

File: lib/cli/scripts/scan-env.ts [hunk 2]
```diff
@@ -262,11 +306,19 @@ function handleError(error) {
     failScript();
 }
 
+/**
+ * Log the error an exit
```
Comment (AleksanderSklorz) [inline lib/cli/scripts/scan-env.ts:310]: probably typo? I guess you meant 'and' not 'an'

File: lib/content-services/src/lib/tag/tag-node-list.component.spec.ts [hunk 3]
```diff
@@ -149,7 +159,12 @@ describe('TagNodeList', () => {
     describe('Limit tags display', () => {
         let initialEntries: TagEntry[];
 
-        async function renderTags(entries?: TagEntry[]): Promise<any> {
+        /**
+         * Renter tags
```
Comment (AleksanderSklorz) [inline lib/content-services/src/lib/tag/tag-node-list.component.spec.ts:163]: typo - should be 'Render'

**Review by nikita-web-ua — APPROVED** [review]

File: .eslintrc.js (hunk 1 above)
Comment (DenysVuika) [reply .eslintrc.js:27]: those functions sometimes also good to document, to give other devs ideas what they are doing, anyway we can review later

File: lib/content-services/src/lib/tag/tag-node-list.component.spec.ts (hunk 3 above)
Comment (DenysVuika) [reply lib/content-services/src/lib/tag/tag-node-list.component.spec.ts]: thanks, fixed

File: lib/cli/scripts/scan-env.ts (hunk 2 above)
Comment (DenysVuika) [reply lib/cli/scripts/scan-env.ts]: fixed

**Review by AleksanderSklorz — APPROVED** [review]

**Pull request merged**